tests/asg/asg_test.go

```diff
@@ -84,7 +86,94 @@ var _ = Describe("{ClusterScaleUpDown}", func() {
 		opts := make(map[string]bool)
 		opts[scheduler.OptionsWaitForResourceLeakCleanup] = true
 		ValidateAndDestroy(contexts, opts)
+	})
+})
+
+// This test randomly kills one volume driver node and ensures cluster remains
+// intact by ASG
+var _ = Describe("{chaosTest}", func() {
```

Comment:
Let's have a more descriptive name as `chaosTest` is pretty generic and a lot of tests could fall under that.

Reply:
Renamed it to `ASGKillRandomNodes`